Lib/contextlib.py

```diff
@@ -102,15 +102,20 @@ def __exit__(self, type, value, traceback):
                 # Suppress StopIteration *unless* it's the same exception that
                 # was passed to throw().  This prevents a StopIteration
                 # raised inside the "with" statement from being suppressed.
-                return exc is not value
+                # Conform to PEP 8 - "Either all return statements in a function
+                # should return an expression, or none of them should".
+                if exc is not value:
```

Comment:
The added comment is also incorrect, as `exc is not value` is already an expression.